### PR TITLE
Update documentation

### PR DESCRIPTION
**Pull request by i-oden**
> **Before submitting the PR, please go through the sections below and fill in what you can. If there are any items that are irrelevant for the current PR, remove the row. If a relevant option is missing, please add it as an item and add a PR comment informing that the new option should be included into this template.**

> **All _relevant_ items should be ticked before the PR is merged**

# Description

- [x] **Summary of the changes and the related issue:** Update of documentation with examples. The branch is called new_python because I started bumping versions.

To test:
1. Run `pip install -r requirements-dev.txt`
2. Go into `docs` directory
3. Run `make html` (or `make clean html` to build from scratch)
4. Run `open _build/html/index.html`

The browser should now open the documentation.

## Type of change

- [x] Documentation
- [x] Security Alert fix
- [x] Workflow

# Checklist:

## General

- [x] [Changelog](../CHANGELOG.md): New row added. Not needed when PR includes _only_ tests.

## Repository / Releases

- [x] Rebase / update of branch done

## Checks

- [x] CodeQL passes
- [x] Formatting: Black & Prettier checks pass
- Tests
  - [x] The tests pass

### CODE REVIEW

**Comment by i-oden** [comment]
@valyo Added a couple more examples, but now I will wait for review.

File: docs/examples.rst [hunk 1]
```diff
@@ -0,0 +1,285 @@
...
+
+
+.. admonition:: Forgotten your...
+
+    * **Username?** Contact support. Changing username or authenticating with email is currently not possible.
+    * **Password?** You can reset your password `here <https://delivery.scilifelab.se/reset_password>`_.
+
+.. danger:: 
+
+    After completing authentication, ``dds-cli`` will automatically save an authentication in the directory where you run the command, unless otherwise specified (see the command documentation). ``dds-cli`` will use this token as a session when you run future commands.
```
Comment (valyo) [inline docs/examples.rst:44]: `... will automatically save an authentication token in the directory where you run the command ...`

and isn't the token created in the home directory?

File: docs/installation.rst [hunk 2]
```diff
@@ -37,160 +46,211 @@ PyPi
...
 Bianca
 -------
 
-.. admonition:: To be aware of
+.. admonition:: Terminology in this section
+
+   * **SENS project** / ``<SENS-project>``: The active SNIC SENS research project, connected to Uppmax. Not a DDS delivery project.
+   * **DDS project** / ``<DDS-project>``: The active DDS delivery project you want to upload data to / download data from or manage. 
+
+.. admonition:: Important
```
[the remark this answers is not in the document]
Comment (i-oden) [reply docs/installation.rst:201]: I added this here. I've also fixed the things you commented on, so it's ready for a new review

**Review by valyo — APPROVED** [review]

File: docs/examples.rst (hunk 1 above)
Comment (i-oden) [reply docs/examples.rst]: > don't you think it is better to add `token file` here?

Was this message actually suppose to be sent yesterday or was it a case of browser cache again? 😅

Comment (valyo) [reply docs/examples.rst]: it was a leftover from before you added the token file text :)

**Pull request merged**